app/views/pages/home.html.erb

```diff
@@ -1,51 +1,105 @@
 <div class="remodal-bg">
-  <div class="signin-section">
-    <div class="container-lg text-center">
-      <ul>
-        <li><%= octicon 'book', height: 38 %></li>
-        <li><%= octicon 'pencil', height: 38 %></li>
-        <li><%= octicon 'mortar-board', height: 38 %></li>
-        <li><%= octicon 'organization', height: 38 %></li>
-        <li><%= octicon 'calendar', height: 38 %></li>
-        <li><%= octicon 'comment-discussion', height: 38 %></li>
-      </ul>
-
-      <h1 class="heading"><%= t('views.pages.banner_heading') %></h1>
-      <p class="subheading"><%= t('views.pages.banner_subheading') %></p>
-
-      <div class="d-flex flex-column flex-items-center">
-        <a class="open-video mb-3" data-remodal-target="getting-started-video"><%= image_tag('play@2x.png', class: 'play-icon', alt: 'Play') %><%= t('views.pages.see_how_it_works') %></a>
-        <%= link_to t('views.pages.sign_in_to_start'), login_path, class: 'btn btn-blue btn-large btn-blue' %>
+  <div class="signin-section py-6 px-1">
+    <div class="container-lg clearfix p-responsive mt-md-4">
+      <h1 class="alt-h1 text-center"><%= t('.signin_section.banner_heading') %></h1>
+      <p class="alt-lead text-center"><%= t('.signin_section.banner_subheading') %></p>
+
+      <div class="clearfix mt-4">
+        <div class="col-md-3 p-3 float-md-left">
+          <%= image_tag('classroom-logo.png', class:"classroom-logo width-fit", alt: 'Classroom Logo') %>
+        </div>
+        <div class="col-md-8 float-left width-fit">
+            <p class="f4 text-gray"><%= t('.signin_section.why_classroom_html') %></p>
+
+            <div class="d-flex flex-column start-container">
+              <%= link_to t('.signin_section.sign_in_to_start'), login_path, class: 'btn btn-large btn-blue mt-1' %>
+            </div>
+        </div>
       </div>
     </div>
   </div>
 
-  <div class="information-section">
-    <div class="container-lg">
-      <div class="columns">
-        <div class="col-md-4 float-left p-4">
-          <h2><%= octicon 'repo', height: 22 %> <%= t('views.pages.create_assignments') %></h2>
-          <p>
-            <%= t('views.pages.create_assignments_desc') %>
-          </p>
+  <div class="why-classroom-section border-top border-gray-light bg-gray-light">
+    <div class="container-lg clearfix p-responsive py-4">
+      <h2 class="alt-h2"><%= t('.why_classroom_section.heading') %></h2>
+      <div class="col-md-11 float-right mt-4">
+        <div class="col-sm-3 mb-4 mx-4 mt-0 float-sm-right">
+          <img class="width-full" src="https://via.placeholder.com/250x200"/>
         </div>
 
-        <div class="col-md-4 float-left p-4">
-          <h2><%= octicon 'organization', height: 22 %> <%= t('views.pages.invite_students') %></h2>
-          <p>
-            <%= t('views.pages.invite_students_desc') %>
-          </p>
+        <% t('.why_classroom_section.items')[0..2].each do |item| %>
```

Comment:
Split this into two sections because of CSS shenangans with floating both the image and Classroom-Desktop card to the right.